xGame/client/src/Components/Margins/Footer.js

```diff
@@ -0,0 +1,14 @@
+import React, {Component} from "react";
```

Comment:
Similarly, this one would just be:

import React from "react";

export default Footer = () => {
	return(
		<p>add footer data</p>
	);
}

Reply:
Ah wait, the "< p >" tags aren't appearing correctly in this comment. Pretend that those tags are still around the "add footer data" part